packages/makeup-roving-tabindex/src/index.js

```diff
@@ -3,41 +3,46 @@
 import * as NavigationEmitter from 'makeup-navigation-emitter';
 
 const defaultOptions = {
-    autoReset: null,
-    index: 0,
+    autoInit: 'interactive',
+    autoReset: 'current',
     wrap: false,
     axis: 'both'
 };
 
-const nodeListToArray = (nodeList) => Array.prototype.slice.call(nodeList);
+function onModelMutation(e) {
+    const { toIndex } = e.detail;
 
```

Comment:
It looks like `onModelMutation`, `onModelInit`, and `onModelReset` all have slightly different logic to do the same thing. I think it would be nice to move this out into a separate function. Something like
```
function refreshTabindex(items, focusIndex) {
  items.forEach((el, i) =>
    el.setAttribute(
      'tabindex',
      i === focusIndex ? '0' : '-1'
    )
  );
}
```

Reply:
Yeah, good idea. Will do. I didn't get to clean up similar events in makeup-active-descendant either. Will try and squeeze that in too.